### ISSUE TITLE
Standardize question paper names

### ISSUE THREAD

**Issue by mukul-mehta** [metakgp/mfqp#53]
I think we could convert paper names to a specific format since there are certain discrepancies when searching for question papers. For example - Algorithms I and CS21003 give different results although they represent the same subject. 
We could use the format YYYY/(Aut/Spring)/(End/Mid)/SubjectCode/SubjectName

Since we already have a table mapping subject names to codes, we could try using regex to match keywords and create a name for most papers if not all.

**Comment by grapheo12** [comment]
Static filws stored in metakgp cdn does not have the subect code in its link. Most of the papers from library site have them. For the former, we need a mapping from Subject name to code. This may require some searching (or if you have them then its good :-)). To add the code either we need to insert that in the subject name itself or create a new field in the entries of `data.json`. This may require change in the fundamental search algorithm.

**Comment by cipherLord** [comment]
Hey @grapheo12 @mukul-mehta, has this issue been resolved ?

**Comment by mukul-mehta** [comment]
@cipherLord Nope, since the paper names aren't really in a well defined format, I don't think this was attempted. I believe with IQPS, we could do this, plus we also have a mapping of subject name to codes. @grapheo12 what do you think?

**Comment by grapheo12** [comment]
Yes we can definitely do this. But just give me time to deploy IQPS first with the current version. :-|
